python/grass/jupyter/interact_display.py

```diff
@@ -70,84 +76,126 @@ def __init__(self, width=400, height=400):
-        return coords_folium[1], coords_folium[0]  # Return Lat and Lon
-
-    def _convert_extent(self, env=None):
-        """This function returns the bounding box of the current region
-        in WGS84, the required projection for folium"""
-
-        # Get proj of current GRASS region
-        proj = gs.read_command("g.proj", flags="jf", env=env)
-        # Get extent
-        extent = gs.parse_command("g.region", flags="g", env=env)
-        # Convert extent to EPSG:3857, required projection for Folium
-        north, east = self._convert_coordinates(extent["e"], extent["n"], proj)
-        south, west = self._convert_coordinates(extent["w"], extent["s"], proj)
-        extent = {"north": north, "south": south, "east": east, "west": west}
-        return extent
-
-    def _folium_bounding_box(self, extent):
-        """Reformats extent into bounding box to pass to folium"""
-        return [[extent["north"], extent["west"]], [extent["south"], extent["east"]]]
 
     def add_vector(self, name):
         """Imports vector into temporary WGS84 location,
         re-formats to a GeoJSON and adds to folium map.
 
-        Arguments:
-            name -- a positional-only parameter; name of vector to be added
-            to map as a string"""
+        :param str name: name of vector to be added to map;
+                         positional-only parameter
+        """
 
         # Find full name of vector
         file_info = gs.find_file(name, element="vector")
         full_name = file_info["fullname"]
         name = file_info["name"]
         # Reproject vector into WGS84 Location
-        env_info = gs.gisenv(env=os.environ)
+        env_info = gs.gisenv(env=self._src_env)
         gs.run_command(
             "v.proj",
             input=full_name,
             location=env_info["LOCATION_NAME"],
             dbase=env_info["GISDBASE"],
-            env=self._vector_env,
+            env=self._wgs84_env,
         )
         # Convert to GeoJSON
-        json_file = self.tmp_dir / f"tmp_{name}.json"
+        json_file = Path(self._tmp_dir.name) / f"tmp_{name}.json"
         gs.run_command(
             "v.out.ogr",
             input=name,
             output=json_file,
             format="GeoJSON",
-            env=self._vector_env,
+            env=self._wgs84_env,
         )
         # Import GeoJSON to folium and add to map
         folium.GeoJson(str(json_file), name=name).add_to(self.map)
 
+    def add_raster(self, name, opacity=0.8):
+        """Imports raster into temporary WGS84 location,
+        exports as png and overlays on folium map
+
+        :param str name: name of raster to add to display; positional-only parameter
+        :param float opacity: raster opacity, number between
+                              0 (transparent) and 1 (opaque)
+        """
+
+        # Find full name of raster
+        file_info = gs.find_file(name, element="raster")
```

Comment:
raster makes sense, but it should be cell here. Find file is a low-level function looking for a file. Hopefully, this will change at some point, but now "raster" generates a verbose message, but "cell" does not. See implementation of find_file.